# Field Review: Single cell technologies in sequence assembly and genome construction
Single cell technologies allow us to inspect biology in unprecedented details and resolution. Instead of treating a tissue or a bacterial colony as a unanimous mass, we can in theory start to understand the roles each cell play in a community. Single cell DNA sequencing methods have been developed in the last few years yet there are still major challenges in correcting the technical errors associated with these technologies using bioinformatics approaches. 

In this field review, focus on explaining these challenges and identify how bioinformatics algorithms deal with noisy single cell sequence data.  Highlight how these algorithms/tools are used for genome assembly and closing in microbes. Describe approaches for combining short- and long- read data for assembly of large eukaryotic genomes.  Also, with the advent of single molecular sequencing (specifically focusing on Oxford Nanopore and 10x), discuss how these “3rd generation” sequencing platforms can be used to move single cell studies forward and what are some of the new bioinformatics tools developed for these sequencing platforms. 

Reference readings to get you started:
- https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4441954/
- https://www.nature.com/articles/nbt.3103
