# Current Paper

## Current Paper: PastML
> Ishikawa, S. A., Zhukova, A., Iwasaki, W. & Gascuel, O. A Fast Likelihood Method to Reconstruct and Visualize Ancestral Scenarios. _Mol. Biol. Evol._ 36, 2069–2085 (2019).
- PastML

## Field Review: Sequence evolution and phylogenetics

The focus of this talk will be on phylogeny construction. You can start by considering what a
phylogeny is (for nucleotide sequences), and how the biological question influences the sequence(s) we choose for phylogeny construction (evolution of particular genes/gene families vs of a particular group of organisms, degree of sequence variation). The focus of this talk will be on two main methods for phylogeny reconstruction – max parsimony and max likelihood. You’ll need to describe the basic principles of both these approaches, the pros and cons of each, and especially, think about how the sequence selection might influence the quality/validity of the tree we reconstruct with either of these approaches. In this process, you’ll also need to think about tree rearrangement methods for finding optimal phylogenetic trees with and without an outgroup sequence (look at branch rearrangements using Nearest Neighbour Interchange (NNI) and Tree Bisection and Reconnection (TBR), and perhaps also the Close Neighbour Interchange distance used in Mega).

### Suggested Readings Chapters 1-3, 8 for background reading 
- http://www.gen-info.osaka-u.ac.jp/~yonishi/bioinfo-new.pdf 

### Parsimony and Likelihood methods
- https://mbe.oxfordjournals.org/content/17/6/839.full
- https://www.ncbi.nlm.nih.gov/pubmed/9501494 

### Tree rearrangements
- https://www.ncbi.nlm.nih.gov/pubmed/8944893 

### Recent paper on a suite of tools related to molecular phylogeny reconstruction, might be interesting to look at (MEGA5):
- https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3203626/

